src/OpenDiffix.Core/Executor.fs

```diff
@@ -55,7 +55,10 @@ let private invokeHooks aggregationContext anonymizationContext hooks buckets =
   match anonymizationContext with
   | None -> buckets
   | Some anonymizationContext ->
-    List.fold (fun buckets hook -> hook aggregationContext anonymizationContext buckets) buckets hooks
+    if aggregationContext.GroupingLabels.Length > 0 then
```

Comment:
The hooks do these checks themselves (at least LED does).

Reply:
Both hooks assert that the `low-count` aggregator is present, which it won't be for global aggregation. This is needed to avoid a crash for such queries.